### PR TITLE
Set up CI with Azure Pipelines

### PR DESCRIPTION
**Pull request by jfversluis**

### CODE REVIEW

File: azure-pipelines.yml [hunk 1]
```diff
@@ -0,0 +1,96 @@
+
+variables:
+  CurrentSemanticVersionBase: '1.5.3'
+  PreviewNumber: $[counter(variables['CurrentSemanticVersionBase'], 1001)]
+  CurrentSemanticVersion: '$(CurrentSemanticVersionBase)-preview$(PreviewNumber)'
+  NugetPackageVersion: '$(CurrentSemanticVersion)'
+  MONO_VERSION: 6_4_0
+  XCODE_VERSION: 11.1
```
Comment (AndreiMisiukevich) [inline azure-pipelines.yml:8]: 11.5 is the current stable version, but I don't know if it's available on Azure

Comment (jfversluis) [reply azure-pipelines.yml:8]: Yep! Don't think so, but I'm still working on this one :)

Comment (jfversluis) [reply azure-pipelines.yml]: Updated to macOS 10.15 and Xcode 11.4 which seems to be the default on the Hosted Mac agents for now

**Comment by jfversluis** [comment]
OK, for I first step I think this is good now. Just CI for now and publishing the NuGets to the Azure DevOps server.

I guess it would be nice to get a nightly build up on GitHub packages here and then whenever we tag something let that go to actual NuGet.

File: azure-pipelines.yml [hunk 2]
```diff
@@ -0,0 +1,141 @@
...
+          inputs:
+            scanType: LogOnly
+        - task: ms.vss-governance-buildtask.governance-build-task-component-detection.ComponentGovernanceComponentDetection@0
+          condition: startsWith(variables['Build.SourceBranch'], 'refs/tags/')
+          displayName: Component Detection - Report
+        - task: securedevelopmentteam.vss-secure-development-tools.build-task-policheck.PoliCheck@1
+          condition: eq(variables['RunPoliCheck'], 'true')
+          displayName: 'PoliCheck'
+          inputs:
+            targetType: F
```
Comment (jfversluis) [inline azure-pipelines.yml:99]: @jamesmontemagno this looked important, is this OK or do I need to do anything else for this?

**Pull request merged**